crates/service-client/src/lambda.rs

```diff
@@ -128,7 +91,11 @@ impl LambdaClient {
         if let Some(profile_name) = profile_name {
             config = config.profile_name(profile_name);
         };
-        config = config.http_client(HyperClientBuilder::new().build(HTTPS_NATIVE_ROOTS.clone()));
+        config = config.http_client(
+            HyperClientBuilder::new()
+                .crypto_mode(CryptoMode::AwsLc)
```

Comment:
I think we need to use `Ring` and not the aws thing, because it breaks with another import where we're forced to use ring